docs/UserGuide.md

```diff
@@ -160,11 +160,10 @@ Format: `exit`
 ### Saving the data
 
 PropertyWhiz's data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
-**TODO**: Change `addressbook.json` filename
 
 ### Editing the data file
 
-PropertyWhiz's data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+PropertyWhiz's data are saved as a JSON file `[JAR file location]/data/propertywhiz.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
I think the section "Archiving data files" (further down) should be removed since it's only part of the original AB3.